.travis.yml

```diff
@@ -4,5 +4,7 @@ python:
 cache: pip
 install:
   - pip install -r requirements.txt
+before_script:
+  - flake8 .
 script:
   - python ./init.py

```

Comment:
I don't believe this should be here, am I correct @janakrajchadha @athityakumar

Reply:
Linter tests usually go into the script section;  usually, any custom changes like setting env is only done in `before_script`.